### PR TITLE
LIB-710: scroll to top when navigating to new page

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: src/theme/DocItem/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, {useEffect} from 'react';
```
Comment (oriooctopus) [inline src/theme/DocItem/index.js]: Before, when navigating to a new page, we wouldn't scroll to the top, so we would just stay at the bottom or the middle

Comment (jjwill) [inline src/theme/DocItem/index.js]: spaces

File: src/theme/DocItem/index.js [hunk 2]
```diff
@@ -14,6 +14,11 @@ import ArrowRight from 'img/shared/arrow-right.svg';
 import classnames from 'classnames';
 import styles from './styles.module.css';
 
+const OVERFLOW_CONTAINER_CLASS = 'nav-pusher';
```
Comment (jjwill) [inline src/theme/DocItem/index.js]: No need for `const` if this isn't used anywhere else.

**Review by jjwill — APPROVED** [review]
LGTM

**Pull request closed without merging**